### PR TITLE
i just changed a bunch of random values

### PR DESCRIPTION
**Pull request by 7ez**

### CODE REVIEW

**Comment by Lxmune** [comment]
hello I approve this change

**Comment by 7ez** [comment]
> hello I approve this change

french detected opinion rejected

File: src/osu_2019/pp.rs [hunk 1]
```diff
@@ -265,7 +265,7 @@ impl<'m> OsuPP<'m> {
 
             if streams_nerf < 1.09 {
                 let acc_factor = (1.0 - self.acc.unwrap()).abs();
-                acc_depression = (0.84 - acc_factor).max(0.5);
+                acc_depression = (0.85 - acc_factor).max(0.5);
```
Comment (7ez) [inline src/osu_2019/pp.rs:268]: idk which way this goes but yeah

**Review by RealistikDash — APPROVED** [review]
he has a gun

**Comment by blobnom** [comment]
🏴

**Comment by 7ez** [comment]
image
fuck him

**Pull request merged**